enderio-base/src/main/java/com/enderio/base/client/renderers/GraveBER.java

```diff
@@ -0,0 +1,57 @@
+package com.enderio.base.client.renderers;
+
+import java.util.Map;
+
+import com.enderio.base.common.blockentity.GraveBE;
+import com.mojang.authlib.GameProfile;
+import com.mojang.authlib.minecraft.MinecraftProfileTexture;
+import com.mojang.authlib.minecraft.MinecraftProfileTexture.Type;
+import com.mojang.blaze3d.vertex.PoseStack;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.model.SkullModel;
+import net.minecraft.client.model.SkullModelBase;
+import net.minecraft.client.model.geom.ModelLayers;
+import net.minecraft.client.renderer.MultiBufferSource;
+import net.minecraft.client.renderer.RenderType;
+import net.minecraft.client.renderer.blockentity.BlockEntityRenderer;
+import net.minecraft.client.renderer.blockentity.BlockEntityRendererProvider;
+import net.minecraft.client.renderer.blockentity.SkullBlockRenderer;
+import net.minecraft.client.resources.DefaultPlayerSkin;
+import net.minecraft.core.Direction;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.level.block.entity.BlockEntity;
+
+//renders grave as a playerskull
+public class GraveBER implements BlockEntityRenderer<BlockEntity>{
```

Comment:
Maybe name this GraveRenderer or something. I'm not personally a fan of acronyms in class names unless seriously necessary